docs/swagger.yaml

```diff
@@ -108,6 +108,41 @@ paths:
                         type: integer
                         description: user id
                         example: 10
+  /auth/invites:
+    get:
+      tags: 
+        - user
+      operationId: inviteStatusOfUser
+      summary: Invite status of a potential user
```

Comment:
@beblicarl the summary should describe what the endpoint does. not sure this captures it

Reply:
Okayyy